enterprise_access/apps/content_assignments/models.py

```diff
@@ -342,6 +346,8 @@ def annotate_dynamic_fields_onto_queryset(cls, queryset):
                 LearnerContentAssignmentAction.objects.filter(
                     assignment=OuterRef('uuid'),
                     action_type=AssignmentActions.REMINDED,
+                    completed_at__isnull=False,
+                    error_reason__isnull=True,
```

Comment:
Another tiniest possible nit: can we switch this so that the error reason filter comes before the completed_at filter?  That way, it'll match the filter order above.